.github/renovate.json5

```
@@ -60,6 +60,15 @@
       separateMajorMinor: true,
       separateMinorPatch: true
     },
+    {
+      // Do not update to patch versions of the Go Toolchain.
+      // Default golang images set the environment variable GOTOOLCHAIN=local
+      // and we don't want to enforce every (test-)image to be on the latest patch level.
+      "matchManagers": ["gomod"],
+      "matchUpdateTypes": ["patch"],
+      "matchPackagePatterns": ["go"],
+      "enabled": false
```

nit, to be consistent with the rest of the properties
```suggestion
      matchManagers: ["gomod"],
      matchUpdateTypes: ["patch"],
      matchPackagePatterns: ["go"],
      enabled: false
```